user/updatePF.php

```diff
@@ -0,0 +1,44 @@
+<?php
+//Maak een class aan
+class UserUpload {
+  public static function run(){
+    session_start();
+
+    //Haal uit de session de gebruikersnaam van de ingelogde gebruiker
+    $user = $_SESSION["username"];
+
+    //Maak willekeurige string
+    $randomstring = bin2hex(random_bytes(30));
```

Comment:
Weet niet of je dit ook specifiek hebt uitgezocht, maar netjes! random_bytes is een safe random functie die daadwerkelijk randomness genereert